int32 id
float32 x
float32 y
bool first_aid